### PR TITLE
Added inline if-else to remove click function and hover pointer class

### PR DESCRIPTION
**Pull request by irisslee**
Fixes ##1552

### What changes did you make?

- Added inline if else to remove click function (one that sorts) and hover pointer class 
-
-

### Why did you make the changes (we will use this info to test)?

- Because the last column shouldn't sort or change pointer on hover
-
-

### Issue-Specific User Account

If you registered a new, temporary TDM User Account for this issue, indicate the
username (i.e., email address) for the account.

### Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

File: client/src/components/Projects/ProjectsPage.js [hunk 1]
```diff
@@ -471,8 +471,14 @@ const ProjectsPage = ({ account, contentContainerRef }) => {
                         return (
                           <td
                             key={i}
-                            className={`${classes.td} ${classes.theadLabel}`}
-                            onClick={() => handleSort(header.id)}
+                            className={
+                              i == 8
```
Comment (ZekeAranyLucas) [inline client/src/components/Projects/ProjectsPage.js:475]: what does 8 mean?

this feels fragile, the design folks keep changing the required columns. consider structuring it to make it clearer why this code is needed.

**Review by ZekeAranyLucas — CHANGES_REQUESTED** [review]
nice getting this in a PR so fast! :)

File: client/src/components/Projects/ProjectsPage.js (hunk 1 above)
Comment (irisslee) [reply client/src/components/Projects/ProjectsPage.js:475]: [https://github.com/hackforla/tdm-calculator/blob/8c0d6bc7fa3b80aa4a7f8b0fcef8a7e93bafd006/client/src/components/Projects/ProjectsPage.js#L366-L385](url)

The function iterates through this list of headers and adds classes and click functions. I'm just using the index on this list of headers. I agree that it's fragile! Another option would be to add conditional based on `label` being blank string, or maybe we can add a boolean property to this header list.... something like - `"sortfunction": true/false` and base the conditionals off of that. I prefer the latter - I think it's more explicit. What do you think?

**Comment by irisslee** [comment]
@entrotech @ZekeAranyLucas  Here are my changes

**Review by ZekeAranyLucas — APPROVED** [review]
much clearer! ship it

**Pull request merged**